standalone/src/data_conversion_layer/start_request_serialization.cpp

```diff
@@ -85,7 +85,9 @@ RawData data_conversion_layer::start_request::serialize(const data_conversion_la
   const uint8_t active_zone_set_enabled{ 0b00001000 };
   const uint8_t io_pin_data_enabled{ 0b00001000 };
   const uint8_t scan_counter_enabled{ 0b00001000 };
-  const uint8_t speed_encoder_enabled{ 0 }; /**< 0000000bin disabled, 00001111bin enabled.*/
+  //const uint8_t speed_encoder_enabled{ 0 }; /**< 0000000bin disabled, 00001111bin enabled.*/
+  const uint8_t speed_encoder_enabled{ static_cast<uint8_t>(
+    msg.master_device_settings_.encoderDataEnabled() ? 0b00001000 : 0b00000000) };
```

Comment:
according to the datalogic manual it should be:
```suggestion
  const uint8_t speed_encoder_enabled{ static_cast<uint8_t>(
    msg.master_device_settings_.encoderDataEnabled() ? 0b00001111 : 0b00000000) };
```